src/pages/meta-chain/index.page.tsx

```diff
@@ -37,20 +39,27 @@ export default function MetaChainPage() {
         />
       </div>
       <MetaChainKeyFeatures />
+      <MetaChainRoadmap roadMapImg={roadMapImg} />
     </>
   );
 }
+async function getRoadMapFromPrismic(): Promise<any> {
+  const endpoint = prismic.createClient("defichain");
```

Comment:
for 3rd party plugin, let's add a try-catch, log the error and return an empty array if error